modules/authentication/proc-provisioning-users-from-microsoft-azure-to-the-software-catalog.adoc

```diff
@@ -0,0 +1,289 @@
+[id="provisioning-users-from-microsoft-azure-to-the-software-catalog"]
+= Provisioning users from Microsoft Azure to the software catalog
```

Comment:
Consider adding an introduction to this section.

Reply:
```suggestion
= Provisioning users from Microsoft Azure to the software catalog

To authenticate users with Microsoft Azure, after xref:enabling-authentication-with-microsoft-azure[Enabling authentication with Microsoft Azure], provision users from Microsoft Azure to the {product-short} software catalog.
```